### PR TITLE
Update documentation for parameters in README

### PR DESCRIPTION
**Pull request by andreaskaris**
Signed-off-by: Andreas Karis <ak.karis@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,7 +105,14 @@ BASE_DOMAIN=your.valid.domain.com
 CLUSTER_NAME=clustername
 # Set your valid DNS VIP, such as 1.1.1.1 for 'ns1.example.com'
 DNS_VIP="1.1.1.1"
+# Set your IP Network stack family, determines the IP address family of the cluster
+# subnet and service subnet, defaults to `v6`
+IP_STACK="v4"
+# Set your host IP Network stack family, determines the IP address family of the provisioning
+# network and external subnets, defaults to the value of `${IP_STACK}`
+HOST_IP_STACK="v4"
```
Comment (hardys) [inline README.md:113]: I think there are some [unresolved issues](https://github.com/openshift-metal3/dev-scripts/pull/1226) with setting HOST_IP_STACK to non-default values so I'm wondering if we should mention it in the README?

File: README.md [hunk 2]
```diff
@@ -105,7 +105,14 @@ BASE_DOMAIN=your.valid.domain.com
...
 # Set your default network type, `OpenShiftSDN` or `OVNKubernetes`, defaults to `OpenShiftSDN`
+# for IPv4, defaults to `OVNKubernetes` for IPv6 and DualStack
```
Comment (hardys) [inline README.md:115]: It might be worth mentioning that ipv6/dual stack will only work with OVNKubernetes?

File: README.md (hunk 1 above)
Comment (andreaskaris) [reply README.md]: Thanks, done.

File: README.md (hunk 2 above)
Comment (andreaskaris) [reply README.md]: Thanks, done, I hope it's better now.

**Comment by derekhiggins** [comment]
/lgtm
thanks

**Comment by hardys** [comment]
/approve 

Thanks!

**Pull request merged**